Makefile

```diff
@@ -102,14 +103,17 @@ staticcheck:
 		`go list ./... | grep -v '\/flux\/stdlib\>'`
 	GO111MODULE=on ./gotool.sh honnef.co/go/tools/cmd/staticcheck ./stdlib/...
 
-test: test-go test-rust
+test: test-go test-rust test-flux
 
 test-go: libflux-go
 	$(GO_TEST) $(GO_TEST_FLAGS) ./...
 
 test-rust:
 	cd libflux && $(CARGO) test $(CARGO_ARGS) && $(CARGO) clippy $(CARGO_ARGS) -- -Dclippy::all
 
+test-flux:
+	$(GO_RUN) ./cmd/flux test
```

Comment:
Do we want a `-v` here so we can see which tests failed in the output?

Reply:
Good idea.